### PR TITLE
Added the admin confirmation password as mentioned in issue #2896

### LINKED ISSUE #2896: Ask for password confirm on voyager:admin command
- Laravel Version: 5.5
- Voyager Version: 1.0
- PHP Version: 7.2

It's a **suggestion not a bug**.

### Description:
I issued the command: `php artisan voyager:admin your@email.com --create` and misspelled my password for example: "passwordù", than when I tried to login did not worked and I had to remove the account and make it again.

It should ask to retype the password so the user can notice errors before creating the account.

### PR DESCRIPTION
**Pull request by swarnava112**
I have added the admin password confirmation when creating an admin from php artisan command.

Closes #2896

### CODE REVIEW

**Comment by emptynick** [comment]
Please don't put comments into your changes if you dont have to e.g. explain a complex condition or something.
Otherwise our code would be 90% comments

File: src/Commands/AdminCommand.php [hunk 1]
```diff
@@ -117,12 +122,19 @@ protected function getUser($create = false)
...
+            $confirmPassword = $this->secret('Confirm Password'); // codegeek2014 #2896
 
             // Ask for email if there wasnt set one
             if (!$email) {
                 $email = $this->ask('Enter the admin email');
             }
 
+            // codegeek2014 #2896
+            if( $password != $confirmPassword){
+                $this->info('Passwords donot match');
```
Comment (emptynick) [inline src/Commands/AdminCommand.php]: Please change this to something like `Passwords don't match`

**Review by emptynick — CHANGES_REQUESTED** [review]

**Comment by swarnava112** [comment]
i have changed the comments.

File: src/Commands/AdminCommand.php [hunk 2]
```diff
@@ -117,12 +122,19 @@ protected function getUser($create = false)
...
+            $confirmPassword = $this->secret('Confirm Password'); 
 
             // Ask for email if there wasnt set one
             if (!$email) {
                 $email = $this->ask('Enter the admin email');
             }
 
+            // passwords donot match
+            if( $password != $confirmPassword){
+                $this->info('Passwords donot match');
```
Comment (emptynick) [inline src/Commands/AdminCommand.php]: Please change this to something like `Passwords don't match`

**Review by emptynick — DISMISSED** [review]

File: src/Commands/AdminCommand.php (hunk 2 above)
Comment (swarnava112) [reply src/Commands/AdminCommand.php]: updated the comment.

**Comment by swarnava112** [comment]
why is the StyleCI check failing? What am I doing wrong?..

**Comment by fletch3555** [comment]
It wants a line before your return.  But you don't need to worry about StyleCI.  It'll fix it itself after this gets merged.

What you do need to do is complete the change requested by @emptynick.  You still have "donot" in your code

File: src/Commands/AdminCommand.php [hunk 3]
```diff
@@ -117,12 +122,19 @@ protected function getUser($create = false)
...
+            $confirmPassword = $this->secret('Confirm Password');
 
             // Ask for email if there wasnt set one
             if (!$email) {
                 $email = $this->ask('Enter the admin email');
             }
 
+            // Passwords don't match
+            if ($password != $confirmPassword) {
+                $this->info('Passwords donot match');
```
Comment (fletch3555) [inline src/Commands/AdminCommand.php]: The text in this line needs to change, as requested by @emptynick

**Review by fletch3555 — DISMISSED** [review]

File: src/Commands/AdminCommand.php [hunk 4]
```diff
@@ -129,9 +129,10 @@ protected function getUser($create = false)
                 $email = $this->ask('Enter the admin email');
             }
 
-            // Passwords don't match
+            // Passwords donot match
```
Comment (swarnava112) [inline src/Commands/AdminCommand.php]: changed don't to donot

File: src/Commands/AdminCommand.php [hunk 5]
```diff
@@ -129,10 +129,9 @@ protected function getUser($create = false)
                 $email = $this->ask('Enter the admin email');
             }
 
-            // Passwords donot match
+            // Passwords don't match
             if ($password != $confirmPassword) {
-                $this->info('Passwords donot match');
-                
+                $this->info("Passwords don't match");
```
Comment (swarnava112) [inline src/Commands/AdminCommand.php]: changed both donot to don't

File: src/Commands/AdminCommand.php [hunk 6]
```diff
@@ -50,6 +50,11 @@ public function handle()
             $this->option('create')
         );
 
+        // the user not returned
+        if (!$user) {
+            exit;
+        }
```
Comment (emptynick) [inline src/Commands/AdminCommand.php:56]: Sorry to bother you again, but what do the above lines do/why are they necessary now?

Comment (swarnava112) [reply src/Commands/AdminCommand.php:56]: if the password doesnot match the a null `$user` is returned, hence if `!$user` then exit. And sorry for the late reply..

Comment (emptynick) [reply src/Commands/AdminCommand.php:56]: Ah yeah, got it

**Review by marktopper — APPROVED** [review]
The entire command could be refactored a bit, but its good for now. Thanks for adding this feature.

**Pull request merged**